src/streams/openssl.c.bak

```diff
@@ -0,0 +1,795 @@
+/*
```

Comment:
This file doesn't look like it belongs here :)

Reply:
Oops.  Thanks - fixed!